### ISSUE TITLE
Keyboard Accessibility

### ISSUE THREAD

**Issue by hdhanani7** [mobify/pinny#30]
- [x] After opening pinny with keyboard, focus should be placed on the first focusable element (header)
- [x] After opening pinny, navigating with the keyboard should ONLY navigate inside the pinny - you should NOT be able to interact with anything outside of the modal (example jQuery implementation can be found [here](http://accessibility.oit.ncsu.edu/blog/2013/09/13/the-incredible-accessible-modal-dialog/))
